Assets/Scripts/Buddy/BuddyStats.cs

```diff
@@ -311,6 +327,18 @@ public void Kill()
 	{
 		_isAlive = false;
 		Destroy( GetComponent<AIController>() );
+		_happiness = 0;
+		RecalculateStat();
+		EmoteDeath();
+		actor.physics.ComeToStop(); // This doesn't quite work and I don't know why
```

Comment:
`ComeToStop()` needs to be called every frame, it doesn't set any internal state for the physics system. You should set something up within the behavior physics controller to actually transition to a state where it just keeps the buddy in place.